src/Components/DisputesTable/GridContent/index.js

```diff
@@ -32,6 +54,14 @@ class GridContent extends Component {
     }
 
     const filteredDisputes = filterFunction(disputes)
+    const indexOfLastRow = currentPage * rowsPerPage
+    const indexOfFirstRow = indexOfLastRow - rowsPerPage
+    const list = filteredDisputes.filter(dispute => {
+      return dispute.arbitratorAddress === process.env.REACT_APP_ARBITRATOR_ADDRESS
+    })
+    const currentRows = list.slice(indexOfFirstRow, indexOfLastRow)
+
+    const totalPages = Math.ceil(list.length / rowsPerPage) || 1
```

Comment:
can you create a function like `const currentRows = (list, indexOfLastRow, indexOfFirstRow)` in the `utils` folder

Reply:
I couldn't find a utils folder. Perhaps you meant "helpers" ?